src/ssh/cockpitsshoptions.c

```diff
@@ -78,40 +75,43 @@ set_environment_bool (gchar **env,
 }
 
 static gboolean
-get_allow_unknown_hosts (gchar **env)
+get_connect_to_unknown_hosts (gchar **env)
 {
   const gchar *remote_peer = g_environ_getenv (env, "COCKPIT_REMOTE_PEER");
 
+  /* Fallback to allowUnknown  */
+  if (!cockpit_conf_string (COCKPIT_CONF_SSH_SECTION, "connectToUnknownHosts") &&
```

Comment:
Why is this cockpit_conf_string? Needs a comment at least. Because we use cockpit_conf_bool on the option below.

Reply:
This is to check whether the option is set at all. _bool cannot tell the difference between "unset" and "set to false".

Reply:
Makes sense. Lets add to the comment there.